.fixtures.yml

```diff
@@ -5,7 +5,7 @@ fixtures:
     facts: 'https://github.com/puppetlabs/puppetlabs-facts.git'
     puppet_agent:
       repo: 'https://github.com/puppetlabs/puppetlabs-puppet_agent.git'
-      ref: v4.13.0
+      ref: v4.19.0
     provision: 'https://github.com/puppetlabs/provision.git'
```

Comment:
IMO this is a really bad design pattern and we should pull from the main branch instead

Reply:
Agreed. It was like that when added in https://github.com/puppetlabs/puppetlabs-apt/commit/43a4d26621e807a1389c18e09d588f54d3f08727 but changed to a version in https://github.com/puppetlabs/puppetlabs-apt/commit/89721b16e7dcfe9cc37e082e63f70e63c7e1f2c6. 🤷

Reply:
Made #1181 to do this.